package.json

```diff
@@ -212,7 +213,7 @@
     "ts-node": "5.0.0",
     "tslint": "5.8.0",
     "tslint-config-prettier": "1.7.0",
-    "typescript": "2.6.2",
+    "typescript": "2.9.2",
```

Comment:
Be careful with upgrading typescript. It has a very close dependency with webpack for compilation (in prod mode).